Backend/prisma/schema.prisma

```diff
@@ -37,33 +37,3 @@ model ServiceProvider {
 
 
 
-
-
-BOOKING TABLE
```

Comment:
@SemanBoon You can disregard my previous comment :) Moving forward let's aim to
- check in clean code off the bat as opposed to cleaning up code in follow up commits
- and have more accurate commit messages (in this case, this commit is actually deleting commented code and not "added database tables for user and service provider")